torch/testing/_internal/schema_check_mode.py

```diff
@@ -47,11 +54,12 @@ def is_mutable(arg_alias_pairs):
             return False
 
         def is_aliasing(output_alias_info, arg_alias_pairs):
-            if output_alias_info is None:
-                return False
             for arg in arg_alias_pairs:
-                if arg.alias_info is not None and bool(len(output_alias_info.after_set & arg.alias_info.after_set)):
-                    return True
+                if arg.alias_info is not None:
+                    if '*' in arg.alias_info.after_set:
```

Comment:
cc @eellison just to make sure this is the right logic for wildcards

Reply:
You still need a check that the types are the same. Wildcard aliases do not alias objects of different type classes. See `alias_analysis.h:40`.

Reply:
The semantics of `chunk(Tensor(a -> *) self, int chunks, int dim=0) -> Tensor(a)[]` is that the input tensor storage will be contained in the output list